src/modules/commander/failsafe/failsafe.h

```diff
@@ -123,6 +123,12 @@ class Failsafe : public FailsafeBase
 		Hold_mode = 2,
 	};
 
+	enum class command_after_high_wind_failsafe : int32_t {
```

Comment:
Ideally I'd like to keep these generic and reuse the same enum for most params where it makes sense.

Reply:
Okay I've align it now with the GF failsafe settings. Does that make sense to you? Or which one would you have re-used?

Reply:
Yes makes sense. I would also make the enum name generic and re-use it.